.rubocop_todo.yml

```diff
@@ -276,109 +263,72 @@ Naming/FileName:
     - 'db/seeds.d/069-provisioning_templates_defaults.rb'
     - 'db/seeds.d/850-global_parameters.rb'
 
-# Offense count: 1
-# This cop supports unsafe autocorrection (--autocorrect-all).
-Performance/InefficientHashSearch:
-  Exclude:
-    - 'db/migrate/20160404130619_remove_satellite_from_notification_name.rb'
-
-# Offense count: 3
-# This cop supports unsafe autocorrection (--autocorrect-all).
+# Offense count: 2
+# Cop supports --auto-correct.
 Rails/ApplicationRecord:
   Exclude:
-    - 'db/migrate/20160404130619_remove_satellite_from_notification_name.rb'
     - 'db/migrate/20161220092005_remove_satellite_from_notification_name_spell_correction.rb'
 
 # Offense count: 2
-# This cop supports unsafe autocorrection (--autocorrect-all).
+# Cop supports --auto-correct.
 # Configuration parameters: Whitelist, AllowedMethods, AllowedReceivers.
-# Whitelist: find_by_sql, find_by_token_for
-# AllowedMethods: find_by_sql, find_by_token_for
-# AllowedReceivers: Gem::Specification, page
+# Whitelist: find_by_sql
+# AllowedMethods: find_by_sql
+# AllowedReceivers: Gem::Specification
 Rails/DynamicFindBy:
   Exclude:
     - 'app/models/concerns/deprecation_notification.rb'
     - 'db/seeds.d/850-global_parameters.rb'
 
-# Offense count: 3
-# This cop supports unsafe autocorrection (--autocorrect-all).
-# Configuration parameters: AllowedMethods, AllowedPatterns.
-# AllowedMethods: order, limit, select, lock
-Rails/FindEach:
-  Exclude:
-    - 'db/migrate/20160404130619_remove_satellite_from_notification_name.rb'
-    - 'db/migrate/20180516103339_update_idm_params.rb'
-
 # Offense count: 3
 Rails/OutputSafety:
   Exclude:
     - 'app/helpers/provisioning_templates_helper_branding.rb'
     - 'app/helpers/rhev_helper.rb'
     - 'app/helpers/theme_application_helper.rb'
 
-# Offense count: 1
-# This cop supports unsafe autocorrection (--autocorrect-all).
+# Offense count: 2
+# Cop supports --auto-correct.
 # Configuration parameters: Include.
 # Include: **/Rakefile, **/*.rake
 Rails/RakeEnvironment:
   Exclude:
     - 'lib/tasks/foreman_theme_satellite_tasks.rake'
 
-# Offense count: 3
-# This cop supports safe autocorrection (--autocorrect).
-# Configuration parameters: EnforcedStyle, Include.
-# SupportedStyles: assert_not, refute
-# Include: **/test/**/*
-Rails/RefuteMethods:
-  Exclude:
-    - 'test/functionals/documentation_links_test.rb'
-
-# Offense count: 2
+# Offense count: 1
 # Configuration parameters: ForbiddenMethods, AllowedMethods.
 # ForbiddenMethods: decrement!, decrement_counter, increment!, increment_counter, insert, insert!, insert_all, insert_all!, toggle!, touch, touch_all, update_all, update_attribute, update_column, update_columns, update_counters, upsert, upsert_all
 Rails/SkipsModelValidations:
   Exclude:
-    - 'db/migrate/20180605150932_deprecate_satellite_templates.rb'
```

Comment:
This seems to make the cop yell again :/

Reply:
Probably not anymore